test/Test/Pos/Ssc/GodTossing/ComputeSharesSpec.hs

```diff
@@ -72,3 +78,19 @@ totalStakeIsZero (getValid -> richmen) =
 invalidStakeErrors :: InvalidRichmenStake GenesisMpcThd -> Bool
 invalidStakeErrors (getInvalid -> richmen) =
     isLeft $ computeShares' richmen
+
+lrcConsistency :: Property
+lrcConsistency =
```

Comment:
No, because it was too much work. I've checked that it fails if I replace `Up` by `Down` in LRC, though (and that should be equivalent).